include/ebpf_structs.h

```diff
@@ -237,10 +247,11 @@ typedef enum bpf_prog_type bpf_prog_type_t;
 // per union member.
 enum bpf_link_type
 {
-    BPF_LINK_TYPE_UNSPEC, ///< Unspecified link type.
-    BPF_LINK_TYPE_PLAIN,  ///< No union members are used in bpf_link_info.
-    BPF_LINK_TYPE_CGROUP, ///< cgroup struct is present in bpf_link_info.
-    BPF_LINK_TYPE_XDP,    ///< xdp struct is present in bpf_link_info.
+    BPF_LINK_TYPE_UNSPEC,   ///< Unspecified link type.
+    BPF_LINK_TYPE_PLAIN,    ///< No union members are used in bpf_link_info.
+    BPF_LINK_TYPE_CGROUP,   ///< cgroup struct is present in bpf_link_info.
+    BPF_LINK_TYPE_XDP,      ///< xdp struct is present in bpf_link_info.
+    BPF_LINK_TYPE_XDP_TEST, ///< xdp struct is present in bpf_link_info.
```

Comment:
We shouldn't have two link type values that use the same union member.  This is not a program type, it's a union discriminator.  Multiple program types can have the same link type if they use the same union member.